templates/product.liquid

```diff
@@ -192,6 +193,7 @@
         catalogId: "{{ product.catalog_id }}",
         categoryId: "{{ product.category_id }}"
     });
+    window.defaultLists = {{ settings.lists | json }};
```

Comment:
Could we avoid of declaring  **defaultLists** in global window variable? I think we can use this approach in the 
 **add to list** popup dialog instead. https://github.com/VirtoCommerce/vc-theme-default/blob/master/assets/js/lists/account-lists.tpl.liquid#L1